### PR TITLE
Adjust styling for 'more' link in content search results

### LINKED ISSUE #2726: "More" links should give specific context (plus styling)
1.  Concern: The "more..." option is not technically a button-within-a-button in the HTML, but it is still reading as such to a screenreader, because the parent item has `role="button"`. I don't know if this is illegal, but it's disconcerting. I don't know what to do about this. 

2. Actionable: When the "more..." link exists, it should include some context for the screenreader. Can do this with `<button aria-describedby="resultLabelID">` which should read something like "more [pause] Page 6". _(edited)_

3. Actionable: The mockups #2683 show lower-case "m" and an underline (I think the expectation was that this would be rendered as a link, not a button). The underline with the expanded letter-spacing of the button looks awkward to me. @ggeisler?

**Current**
Screen Shot 2019-06-14 at 3 55 50 PM
**Proposed**
Screen Shot 2019-06-14 at 3 50 41 PM
  - remove the underline
  - force the height of `<span class="MuiButton-label">` to `1.5em` so it doesn't make the last line of text hang lower than it's meant to
  - lower-case the "m"

### PR DESCRIPTION
**Pull request by camillevilla**
Closes #2726
This PR makes slight adjustments to the styling of the "More" link in content search results. @jvine, I wasn't sure if you meant a `height` or `line-height` of `1.5em`. Take a look in netlify and let me know if this works. 

## Before
links underlined

## After
linked lower cased with line removed

### CODE REVIEW

**Comment by jvine** [comment]
Huh. Turns out setting the height screws up the focus bubble thingie and makes it hard to see when the button has focus. Setting line-height keeps the bubble.

**Comment by camillevilla** [comment]
Thanks! I've corrected this PR. For anyone else who's curious, the difference between `height` and `line-height` is visible when using keyboard navigation. The GIFs below show how the Material UI ripple animation is cut off by `height` but remains the same with `line-height`. 

#### height: 1.5m
height-example

#### line-height 1.5em
line-height-example

File: src/components/SearchHit.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ export class SearchHit extends Component {
           <SanitizedHtml ruleSet="iiif" htmlString={truncatedHit.after} />
           {' '}
           { truncated && !focused && (
-            <Button className={classes.inlineButton} onClick={showDetails} color="secondary" size="small">
+            <Button className={classes.inlineButton} onClick={showDetails} color="secondary" size="small" aria-describedby={canvasLabel}>
```
Comment (mejackreed) [inline src/components/SearchHit.js]: Should the `aria-describedby` reference an `ID` here?

https://www.w3.org/TR/wai-aria/#aria-describedby

Comment (camillevilla) [reply src/components/SearchHit.js]: 😅 alrighty @mejackreed, let me know what you think about this approach. @cbeer suggested I use companion window IDs to deal with the case where there are multiple content search windows open.

Comment (mejackreed) [reply src/components/SearchHit.js]: Looks good!

**Pull request merged**